.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: DotNet Core org issues
+    url: https://github.com/dotnet/runtime#filing-issues
```

Comment:
I was going to suggest having a .NET Core SDK link here too, but it's linked from this page, so that is probably good enough. @kathleendollard may feel otherwise.